### PR TITLE
Codespaces

### PR DESCRIPTION
**Pull request by tpmccallum**
This is a PR that implements Codespaces.

### CODE REVIEW

File: .devcontainer/bart/install.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+VERSION="v0.6.0"
+# OSARCH="linux-amd64"
```
Comment (tpmccallum) [inline .devcontainer/bart/install.sh:4]:
```suggestion
OSARCH="linux-amd64"
```

File: .devcontainer/bart/install.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+wget https://github.com/fermyon/bartholomew/releases/download/${VERSION}/bart
```
Comment (tpmccallum) [inline .devcontainer/bart/install.sh:6]:
```suggestion
wget https://github.com/fermyon/bartholomew/releases/download/${VERSION}/bart-cli-${VERSION}-${OSARCH}.tar.gz
tar -zxvf bart-cli-${VERSION}-${OSARCH}.tar.gz
```

**Review by tpmccallum — COMMENTED** [review]
Update to use tar files, like Spin does

File: .devcontainer/bart/install.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+VERSION="v0.6.0"
+OSARCH="linux-amd64"
+
+wget https://github.com/fermyon/bartholomew/releases/download/${VERSION}/bart-cli-${VERSION}-${OSARCH}.tar.gz
+tar -zxvf bart-cli-${VERSION}-${OSARCH}.tar.gz
+su
+mv ./bart /usr/local/bin/
+chmod a+x /usr/local/bin/bart
```
Comment (tpmccallum) [inline .devcontainer/bart/install.sh:10]:
```suggestion
```

File: .devcontainer/spin/install.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+VERSION="v0.6.0"
```
Comment (radu-matei) [inline .devcontainer/spin/install.sh:3]: Should this use the Spin install script instead from https://developer.fermyon.com/spin/install?

Comment (tpmccallum) [reply .devcontainer/spin/install.sh:3]: That's a really great point @radu-matei I will try that out.

**Comment by tpmccallum** [comment]
I would love to test this now; contributing to docs is a great use of Codespaces (minimal infrastructure and relatively quick build for end users).

**Comment by tpmccallum** [comment]
Closed this in favour of using cloud preview mode.

**Pull request closed without merging**